odxtools/parentref.py

```diff
@@ -27,6 +26,10 @@ class ParentRef:
     def parent_layer(self) -> "DiagLayer":
         return self._parent_layer
 
+    @property
+    def parent_layer_type(self) -> DiagLayerType:
```

Comment:
is this property really needed? parent_layer is available after all

Reply:
correct. removed...